hugolib/site_render.go

```diff
@@ -130,6 +130,10 @@ func (s *Site) renderPaginator(p *Page) error {
 
 func (s *Site) renderRSS(p *Page) error {
 
+	if !s.isEnabled(kindRSS) {
+		return nil
+	}
+
 	if s.Cfg.GetBool("disableRSS") {
```

Comment:
OK, I understand -- and sure it can, but that is out of scope of this PR -- it would require lots of other test cases. That flag should probably be removed at some point.